controllers/twophase/twophase_suite_test.go

```diff
@@ -94,10 +94,16 @@ func (in *fakeTwoPhaseChaos) GetStatus() *v1alpha1.ChaosStatus {
 	return &in.Status
 }
 
+// IsDeleted returns whether this resource has been deleted
 func (in *fakeTwoPhaseChaos) IsDeleted() bool {
 	return in.Deleted
 }
 
+// IsPaused returns whether this resource is paused
```

Comment:
`whether this resource is paused` to `whether this resource has been paused` ? To keep same doc style.